charts/kube-prometheus-stack/templates/prometheus-operator/deployment.yaml

```diff
@@ -63,6 +63,12 @@ spec:
             - --kubelet-selector={{ .Values.prometheusOperator.kubeletService.selector }}
             {{- end }}
             {{- end }}
+            {{- if .Values.prometheusOperator.kubeletEndpointsEnabled }}
```

Comment:
if in combination with an boolean might be mis-leading.

Since `--kubelet-endpoints=` is set to true by default, I set the value to `false` keep the flag still to true, because `--kubelet-endpoints=false` would be omit.

not sure, if

`{{- if (print .Values.prometheusOperator.kubeletEndpointsEnabled) }}`

would help here, since it converts the value to a string and `"false"` is still true-ish, I guess?

Reply:
Good point, maybe just omit the ifs entirely and go by the values.yaml values?